feat: fcc-todo-app (#51367)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-localstorage-by-building-a-todo/64ec9b10356c2d8aa05d9ce1.md

---
id: 64ec9b10356c2d8aa05d9ce1
title: Step 23
challengeType: 0
dashedName: step-23
---

# --description--

> After adding the task to the page, you need to close the form modal so you can see the task. To do that, use `classList` to toggle the class `hidden`on `taskForm` element.

Reviewer: ```suggestion
After adding the task to the page, you should close the form modal to view the task. To do this, utilize `classList` to toggle the `hidden` class on the `taskForm` element.
```
